wordpress-mobile/WordPress-Android
# Minified widget configuration

This PR adds a new dialog fragment to select a "Type" for the minified widget (views/visitors/comments/likes). This field is mandatory in the configuration. The Widget itself is not yet implemented.

To test:
* Add the new widget
* Check that there is a "Type" option on the configuration screen
* Check that you can select a "Type" 
* Selecting a "Type" and a "Site" enables the "Add" button

Update release notes:
